core/src/banking_stage.rs

```diff
@@ -51,6 +49,10 @@ pub const NUM_THREADS: u32 = 4;
 
 const TOTAL_BUFFERED_PACKETS: usize = 500_000;
 
+/// This is used for batching of transactions to process and record.
+/// (700K TPS ~= 70K Tx per Tick spread over NUM_THREADS)
+const MAX_NUM_TRANSACTIONS_PER_BATCH: usize = 64 * 1024 / NUM_THREADS as usize;
```

Comment:
Should this calculation include TICKS_PER_SECOND

Reply:
64k is closest power of 2 to 70k